.github/workflows/linters.yml

```diff
@@ -0,0 +1,70 @@
+name: Linters
+
+on: pull_request
+
```

Comment:
### Your Readme File
![image](https://user-images.githubusercontent.com/93039359/198582988-7deac5ed-702f-48c7-8500-5aa2e8af2c96.png)

- [ ] Your readme file is not descriptive enough. You have added some details but not in a proper way. Kindly add this [code](https://raw.githubusercontent.com/microverseinc/readme-template/master/README.md) in your readme file :+1: 
- [ ] After adding that code kindly replace your own details in the template like Project Name to Capstone or whatever. I guess your understood :blush:
- [ ] You didn't add a 5 min introduction video of your application which is a project requirement. Kindly add it :+1:
- [ ] Kindly add your application deployment link in your readme file to make your application easily accessible for even not tech people. :+1: